.circleci/config.yml

```diff
@@ -60,6 +60,7 @@ jobs:
   scripted3_jdk:
     docker:
       - image: cimg/openjdk:11.0-node
+    resource_class: medium+
```

Comment:
The CI step `scripted3_jdk` kept being killed and it seems like it is because of an out of memory exception. Memory usage was reaching 100%.

I've checked a couple of runs of `scripted3_jdk` on other branches, and they went to 98% memory usage, so it seems the runner was utilized to the max 💪 

Maybe something about this change causes more memory to be used, I'm not sure why though. Making this private forces the compiler to hold more stuff in scope? 🤷 

The `medium+` resource class is included on free plans, but it burns
15 credits a minute over 10/minute for the medium class, [Pricing info here](https://circleci.com/product/features/resource-classes/). It has 6GB of ram, an increase of 2GB.

I don't know what plan the project is running on, and I hope this doesn't incur unwarranted cost.

Reply:
Ah yeah, I fixed this in another PR that I haven't merged yet (I even went on using `large` instances). This is definitely not caused by your change. Thanks!